wp-content/themes/pub/wporg-learn-2020/css/objects/_prism.scss

```diff
@@ -217,7 +217,7 @@ pre.line-numbers > code {
 .wp-code-block-button-container {
 	display: flex;
 	justify-content: right;
-	padding: 1rem;
+	padding: 0.5rem;
```

Comment:
This all looks good, but I think it'd be ideal to leave the original values, and put the changes at the buttom in the "Learn customizations" section. That'll make it easier to sync w/ the original in the future, if we ever want to.

Reply:
Good point, thanks @iandunn moved the Learn specific additions to the bottom of the file.